exasol/analytics/query_handler/query/select.py

```diff
@@ -24,3 +28,46 @@ def __init__(self, query_string: str, output_columns: List[Column]):
     @property
     def output_columns(self) -> List[Column]:
         return self._output_columns
+
+
+class AuditData:
+    """
+    This is a collection of data for auditing. The data represent one audit message.
+    The items in the dictionary correspond to the columns in the audit table.
+    Components at different levels in the call stack can add their own items here.
```

Comment:
Regarding this point, I thought a bit, and I got a question for us, should we make the dict append only, meaning that higher levels in the call stack can only add new entries to the dict, but can't change already set ones. My first impression is, that would reduce surprises. What do you think?